Pipfile

```diff
@@ -72,9 +72,10 @@ google-cloud-tasks = "*"
 simplejson = "*"
 markupsafe = "*"
 pdfkit = "*"
+typing-extensions = "*"
```

Comment:
This is a dev dependency. Is it needed?.

Reply:
GIthub actions was failing without me adding this. I will re-try without it and see if this is needed or not.

Reply:
Removed `typing-extensions` and GA are passing 👍